api/src/main/java/org/apache/iceberg/PartitionSpec.java

```diff
@@ -109,7 +109,10 @@ public int specId() {
       PartitionField field = fields[i];
       Type sourceType = schema.findType(field.sourceId());
       Type resultType = field.transform().getResultType(sourceType);
-      // assign ids for partition fields starting at 100 to leave room for data file's other fields
+      /**
+       * assign ids for partition fields starting at {@link PARTITION_DATA_ID_START}
+       * to leave room for data file's other fields
+       */
```

Comment:
The content looks fine. Please change this to use line comments (`//`) instead of block comments because this is in the middle of code. Also, please remove the javadoc formatting that isn't necessary.

Reply:
@rdblue Thanks for your comments! Those are addressed. Please review~~